### DISCUSSION TITLE
Brython Logo

### DISCUSSION THREAD

**Discussion by shark-in-a-hat** [brython-dev/brython#2357] [Ideas]
I'd like to add a Brython logo to some of the things that I do, but unfortunately the current logo is... um... there is no logo. Not really.  
So I made one:
![brython_sml](https://github.com/brython-dev/brython/assets/107478567/93ee71b1-1f55-4cfe-ba80-364b1b2dce7f)
It's not super good, but it's as good as I can make it. Maybe it will inspire someone to make a better one? Who knows?

If you find it useful - it's free to use, CC0, copyleft. I also have one that's 4x bigger (not better, just 4x resolution) if anyone wants to use it.

**Comment by dunossauro** [comment]
OMG, I love it :heart_eyes:

**Comment by PierreQuentel** [comment]
Thanks for the suggestion @shark-in-a-hat, but I'm afraid I'm lukewarm with this logo.

There is indeed one:

![favicon](https://github.com/brython-dev/brython/assets/3447215/20f91b59-65a4-4a58-b4c9-2e6122c4858a)

the "B" letter in a Celtic font. This is important to me because Brython, besides its meaning "Browser Python", is related to Celtic languages, and I am a Breton speaker and teacher...

**Reply by shark-in-a-hat** [reply]
Fair enough.
Idea nr2:
Here's a B with a Celtic motive, but it's also in a JS/HTML5 shield-logo-thing and in Python like color scheme: 
![b_js](https://github.com/brython-dev/brython/assets/107478567/798ff109-1cd7-40e8-9291-1986c0527e0a)

**Reply by BobSquarePants** [reply]
nice @shark-in-a-hat  !

@PierreQuentel  what about the current letter ![Brython logo](https://private-user-images.githubusercontent.com/3447215/304236829-20f91b59-65a4-4a58-b4c9-2e6122c4858a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM4OTAwMDAsIm5iZiI6MTcxMzg4OTcwMCwicGF0aCI6Ii8zNDQ3MjE1LzMwNDIzNjgyOS0yMGY5MWI1OS02NWE0LTRhNTgtYjRjOS0yZTYxMjJjNDg1OGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MjNUMTYyODIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGJmYTAzYjYzNDQxYTllYTU4YmQ4NTYzOTU3NWNiMmQ0NTY1MjgxMGRmMGMyOTRiZjIyODJjZWNhMGRkYTk3YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.8tA2VbsaPCn-7r1Av6Krc5mX8UUP9l-zTO8vyM29srA) and the Python snake circle and/or pass trough the holes of The B letter ?
